<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_force</class>
 <widget class="QDialog" name="dlg_force">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>529</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Force Feedback Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Disable Forces</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_pCheckTransX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Translational Forces (X-Axis)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_pCheckTransY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Translational Forces (Y-Axis)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_pCheckTransZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Translational Forces (Z-Axis)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="m_pCheckRotX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Rotational Forces (X-Axis)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="m_pCheckRotY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Rotational Forces (Y-Axis)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="m_pCheckRotZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable Rotational Forces (Z-Axis)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="m_pCheckNegButton">
        <property name="toolTip">
         <string>If checked no force is generated if the button is pressed</string>
        </property>
        <property name="text">
         <string>Switch Button Logic</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="TextLabel2_3">
        <property name="text">
         <string>Scale Translations:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="3">
       <widget class="QSlider" name="m_pSliderScaleTranslation">
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>Scale CC Forces:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="3">
       <widget class="QSlider" name="m_pSliderScaleCorrForce">
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="TextLabel2_2">
        <property name="text">
         <string>Scale Steric Forces:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="3">
       <widget class="QSlider" name="m_pSliderScaleStericForce">
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Central Sphere:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="3">
       <widget class="QSlider" name="m_pSliderScaleCentralSphere">
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>Scale Arrow:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="3">
       <widget class="QSlider" name="m_pSliderScaleForceArrow">
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="8">
       <widget class="QLabel" name="StericInteractionThresholds">
        <property name="text">
         <string>Steric Interaction Thresholds (in Angstrom):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="GoodThreshold">
        <property name="text">
         <string>Good:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pEditStericGoodThreshold">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="8" column="4" colspan="2">
       <widget class="QLabel" name="ClashThreshold">
        <property name="text">
         <string>Clash:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="5">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Central Sphere Color Function:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Red</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="m_pSpinSphereColorFunctionRed">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="3" colspan="2">
       <widget class="QSpinBox" name="m_pSpinSphereColorFunctionBlue">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="11" column="5" colspan="2">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="7">
       <widget class="QSpinBox" name="m_pSpinSphereColorFunctionGreen">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="8" column="6" colspan="2">
       <widget class="QLineEdit" name="m_pEditStericClashThreshold">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="4">
       <widget class="QCheckBox" name="m_pCheckForceArrow">
        <property name="toolTip">
         <string>If checked no force is generated if the button is pressed</string>
        </property>
        <property name="text">
         <string>Force Arrow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
